tests/qir/test_pytket_qir_14.ll

```diff
@@ -18,9 +18,9 @@ entry:
   %1 = call i1* @create_creg(i64 10)
   %2 = call i1* @create_creg(i64 10)
   %3 = call i1* @create_creg(i64 9)
-  %4 = call i1* @create_creg(i64 32)
-  %5 = call i1* @create_creg(i64 32)
-  %6 = call i1* @create_creg(i64 32)
+  %4 = call i1* @create_creg(i64 64)
```

Comment:
Is this expected because of the changes in pytket, I am not really sure about it?